src/backend/postgres/query.rs

```diff
@@ -92,4 +92,20 @@ impl QueryBuilder for PostgresQueryBuilder {
             _ => QueryBuilder::prepare_simple_expr_common(self, simple_expr, sql, collector),
         }
     }
+
+    fn prepare_order_expr(
+        &self,
+        order_expr: &OrderExpr,
+        sql: &mut SqlWriter,
+        collector: &mut dyn FnMut(Value),
+    ) {
+        self.prepare_simple_expr(&order_expr.expr, sql, collector);
+        write!(sql, " ").unwrap();
+        self.prepare_order(&order_expr.order, sql, collector);
+        match order_expr.nulls {
+            None => (),
+            Some(Nulls::Last) => write!(sql, " NULLS LAST").unwrap(),
+            Some(Nulls::First) => write!(sql, " NULLS FISRT").unwrap(),
```

Comment:
Just discovered there is a typo, "FISRT".
Check the fix https://github.com/SeaQL/sea-query/commit/ce52bda96653b61d07c2cc9ebaf106169727a160

Reply:
Oh! Sorry for the stupid mistakes. 
fixed.